pkg/virt-controller/watch/export/export.go

```diff
@@ -1021,9 +1033,16 @@ func buildPemFromCert(cert *x509.Certificate) string {
 	return strings.TrimSpace(pemOut.String())
 }
 
+func (ctrl *VMExportController) matchesOrWildCard(hostName, compare string) bool {
+	wildCard := fmt.Sprintf("*%s", getDomainFromHost(hostName, ctrl.KubevirtNamespace))
```

Comment:
I think it would be better for `getDomainNameFromHost()` to not return the preceding '.' and include it in the sprintf call.

Minor quibble

Reply:
Done